py/code_intelligence/graphql.py

```diff
@@ -9,10 +9,10 @@ class GraphQLClient(object):
   """A graphql client for GitHub"""
 
   def __init__(self):
-    if not os.getenv("GITHUB_TOKEN"):
-      raise ValueError("GITHUB_TOKEN must be present as an environment variable upon instantiating this object.")
+    if not os.getenv("INPUT_PERSONAL_ACCESS_TOKEN"):
```

Comment:
Why do we need to change the environment variable? Is it not possible to set the environment variable in the workflow?

Reply:
The problem is that `GITHUB_TOKEN` is a special environment variable which you cannot override in Actions